dissect/target/target.py

```diff
@@ -285,10 +281,11 @@ def _find(find_path: Path):
                         # Attempt to load the target using this loader
                         target = cls._load(sub_entry, ldr)
                         loaded = True
+                        at_least_one_loaded = True
                         yield target
-                    except Exception as e:
-                        getlogger(sub_entry).error("Failed to load target with loader %s", ldr, exc_info=e)
-                        continue
+
+                    except Exception as error:
+                        getlogger(sub_entry).error("Failed to load target with loader %s", ldr, exc_info=error)
```

Comment:
You could argue that the renaming of `e` to `error` doesn't add much value here, and this change could be reduced to just the removal of the `continue` statement. It's also (currently) less consistent with similar code constructs.

Reply:
Yes, agreed, it does not add much value. Changed.